hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -2908,11 +2896,9 @@ public boolean processReport(final DatanodeID nodeID,
 
     if (blockLog.isDebugEnabled()) {
       for (Block b : invalidatedBlocks) {
-        if (blockLog.isDebugEnabled()) {
-          blockLog.debug("BLOCK* processReport 0x{} with lease ID 0x{}: {} on node {} size {} " +
-                          "does not belong to any file.", strBlockReportId, fullBrLeaseId, b,
-                           node, b.getNumBytes());
-        }
+        blockLog.debug("BLOCK* processReport 0x{} with lease ID 0x{}: {} on node {} size {} " +
```

Comment:
Don't we want to keep the check?

Reply:
There is already a judgment, so we can remove the internal redundant judgment.
```
if (blockLog.isDebugEnabled()) {
   for (Block b : invalidatedBlocks) {
      blockLog.debug("BLOCK* processReport 0x{} with lease ID 0x{}: {} on node {} size {} " +
                "does not belong to any file.", strBlockReportId, fullBrLeaseId, b,
          node, b.getNumBytes());
    }
}
```